felix/bpf/proxy/kube-proxy.go

```diff
@@ -215,17 +247,14 @@ func (kp *KubeProxy) OnHostIPsUpdate(IPs []net.IP) {
 }
 
 // OnRouteUpdate should be used to update the internal state of routing tables
-func (kp *KubeProxy) OnRouteUpdate(k routes.Key, v routes.Value) {
-	if err := kp.rt.Update(k, v); err != nil {
-		log.WithField("error", err).Error("kube-proxy: OnRouteUpdate")
-	} else {
-		log.WithFields(log.Fields{"key": k, "value": v}).Debug("kube-proxy: OnRouteUpdate")
-	}
+func (kp *KubeProxy) OnRouteUpdate(k routes.KeyInterface, v routes.ValueInterface) {
+	kp.rt.Update(k, v)
```

Comment:
Is this putting both IPv4 and IPv6 in the same Trie? I'm not sure if the CIDR trie is robust to that; there are code paths that panic if they have to compare v4 to v6 IPs so it may only blow up if you've got IPv4 and IPv6s that start with the same prefix!